### ISSUE TITLE
Welcome Bot

### ISSUE THREAD

**Issue by ruqqq** [SGTinkers/msocietybot#17]
A bot which welcomes new user to the group with a changeable text template.

- Send message to group when new user joined
- Delete old welcome messages whenever a new one is created
- User required to reply (not necessarily reply to the message) within 3 days, else will be kicked
- Use Stage/Scene to modify welcome text for a chat group from a private chat with bot (use inline options and etc.)
- Support trivial templating in welcome message

**Comment by xModulus** [comment]
> Delete old welcome messages whenever a new one is created

What does this mean? Delete old messages that were sent?

> User required to reply (not necessarily reply to the message) within 3 days, else will be kicked

Should we consider having a public lobby and the main group private (or at least prevent people from inviting directly)? The bot moves you to the main group when you satisfy certain conditions (i.e. real human not spam bot).

**Comment by ruqqq** [comment]
> > Delete old welcome messages whenever a new one is created
> 
> What does this mean? Delete old messages that were sent?

Yes, delete the other welcome messages posted previously from the chat. (Good to have, not must have)

> > User required to reply (not necessarily reply to the message) within 3 days, else will be kicked
> 
> Should we consider having a public lobby and the main group private (or at least prevent people from inviting directly)? The bot moves you to the main group when you satisfy certain conditions (i.e. real human not spam bot).

Yes, agreed. As previously discussed.
